lms/djangoapps/bulk_email/tasks.py

```diff
@@ -418,7 +418,22 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
         'failed' count above.
     """
     # Get information from current task's request:
+    parent_task_id = InstructorTask.objects.get(pk=entry_id).task_id
     task_id = subtask_status.task_id
+    total_recipients = len(to_list)
+    recipient_no = 0
+    total_recipients_successful = 0
+    total_recipients_failed = 0
+    recipients_info = {}
+    duplicate_recipients = set()
+
+    log.info("=============== start logging for send email task ===============")
+    log.info(
+        "Task: %s, SubTask: %s, EmailId: %s, TotalRecipients: %s",
+        parent_task_id,
+        task_id, email_id,
```

Comment:
We won't map these to individual people--it's just general information about each task as a whole. But, later on, we also have a log for each individual email that's sent out that we can link to specific people.